examples/README.md

```diff
@@ -1,39 +1,8 @@
 Viewing Examples
 ----------------
 
-First, make sure you have IPython version 1.0 or later installed. Then, to view
-these examples, navigate to this directory and run:
+First, make sure you have [Jupyter](http://jupyter.org/install) installed.
+Then, to view these examples, navigate to this directory and run:
 ```
-ipython notebook --pylab=inline
+jupyter notebook
 ```
-
-Eventually, we'll turn these into HTML (very easy with `ipython nbconvert`) and
-host them online.
-
-Testing Examples
```

Comment:
I'm sure the script I setup before is now broken, but it would be worth trying out something new, maybe nbval:
https://github.com/computationalmodelling/nbval